src/Schema/Events/Tap.ts

```diff
@@ -327,6 +327,29 @@ export interface TappedFairGroup extends TappedEntityGroup {
   action: ActionType.tappedFairGroup
 }
 
+/**
+ * A user taps a grouped card (Explore By, Discover Something new)
+ *
+ * This schema describes events sent to Segment from [[tappedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedCardGroup",
+ *    context_module: "exploreBySection" | discoverSomethingNewSection,
```

Comment:
**suggestion**: I'd pick one of these strings for the example and drop the other as it becomes invalid JSON.